layouts/default.vue

```diff
@@ -152,7 +152,7 @@ export default Vue.extend({
         }
       ];
     },
-    configItems() {
+    configItems(): Array<Record<string, string>> {
```

Comment:
This should be a custom type

Reply:
@Camc314 Created ``LayoutButton`` type for this.